test/Microsoft.ML.Tests/Transformers/ConcatTests.cs

```diff
@@ -62,7 +62,7 @@ ColumnType GetType(Schema schema, string name)
             t = GetType(data.Schema, "f4");
             Assert.True(t is VectorType vt4 && vt4.ItemType == NumberType.R4 && vt4.Size == 0);
 
-            data = ColumnSelectingTransformer.CreateKeep(Env, data, new[] { "f1", "f2", "f3", "f4" });
+            data = ML.Transforms.SelectColumns(new[] { "f1", "f2", "f3", "f4" }).Fit(data).Transform(data);
```

Comment:
>SelectColumns [](start = 33, length = 13)

I've tried to find in this PR source for this extension, and fail.
I know you have it something SelectColumns(string[] keep, bool keepHidden = false);
Can C# allow you to have second method which would look `SelectColumns(params string[] keepColumns)`?
It would be so much nicer to just type columns without any array allocation.

Reply:
Will try to add this, it's a very good suggestion! Thank you

---
In reply to: [254139581](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254139581) [](ancestors = 254139581)

Reply:
I'm not saying you have to update all other SelectColumns calls to stop using arrays and just pass them via params, but it would be nice :)

---
In reply to: [254139904](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254139904) [](ancestors = 254139904,254139581)